internal/hcs/schema2/registry_hive.go

```diff
@@ -0,0 +1,13 @@
+package hcsschema
+
+// NOTE: manually added
+
+type RegistryHive string
+
+// List of RegistryHive
+const (
```

Comment:
These strings are also used in a lot of places under `internal/wclayer`, can we use these constants there too?

Reply:
does wclayer rely mostly on HCS, or does it use other Win32 APIs?
I am seeing a `"DEFAULT"` and also `"*_BASE"` and `"*_DELTA"` hives, so I don't know if we should leverage HCS constants for non-HCS code paths